tf_encrypted/protocol/pond.py

```diff
@@ -989,6 +989,71 @@ def dispatch(self, base_name, *args, container=None, **kwargs):
         else:
             raise TypeError("Don't know how to {}: {}".format(base_name, [type(arg) for arg in args]))
 
+    @memoize
+    def zeros(
```

Comment:
I think we'll want to implement the zeros functions for public, private and masked tensors. Not sure if we need to for pad though might be able to infer the correct type based on whatever is passed in and then used in zeros.

Reply:
Hey Justin, good point. I have added support for public and masked for zeros. Let me know if that's what you had in mind. Thank you!